### PR TITLE
LPD-23462 Automate test of Release Docker images

### PR DESCRIPTION
**Pull request by lucasmiranda0**
Related issue: [LPD-23462](https://liferay.atlassian.net/browse/LPD-23462)
Changes asked here: #9

### CODE REVIEW

File: playwright-dependencies/tests/smoke.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  await page.getByLabel('Password', { exact: true }).click();
+  await page.getByLabel('Password', { exact: true }).fill('new-password');
+  await page.getByLabel('Password', { exact: true }).press('Tab');
+  await page.getByLabel('Reenter Password').fill('new-password');
+  await page.getByLabel('Reenter Password').press('Tab');
+  await page.getByRole('button', { name: 'Save' }).press('Enter');
+
+  const responsePromise = page.waitForResponse('**/o/com-liferay-enterprise-product-notification-web/confirm/');
+  await page.getByRole('button', { name: 'Done' }).click();
+  const response = await responsePromise;
```
Comment (natocesarrego) [inline playwright-dependencies/tests/smoke.spec.ts:28]: @lucasmiranda0, can we do the following?
```
await page.waitForResponse('**/o/com-liferay-enterprise-product-notification-web/confirm/');
await page.getByRole('button', { name: 'Done' }).click();
```

The variable `response` seems to be unnecessary, am I right?

File: playwright-dependencies/tests/smoke.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  await page.getByLabel('Test Test User Profile').click();
+  await page.getByRole('menuitem', { name: 'Account Settings' }).click();
+
+  await page.getByRole('link', { name: 'Password' }).click();
+  await page.getByLabel('Current Password Required').click();
+  await page.getByLabel('Current Password Required').fill('new-password');
+
+  await page.getByLabel('New Password Required').click();
+  await page.getByLabel('New Password Required').fill('new-password-1');
```
Comment (natocesarrego) [inline playwright-dependencies/tests/smoke.spec.ts:38]: @lucasmiranda0, can you change to `new-password` and `new-password-1` to `password` and `new-password`, respectively?

Using magic numbers, like in `new-password-1`, isn't so meaningful :slightly_smiling_face:

File: test_image.sh [hunk 3]
```diff
@@ -223,6 +224,28 @@ function test_docker_image_scripts_2 {
...
+		echo "Directory playwright-dependencies does not exist"
+
+		exit 1
+	fi
+
+	cd playwright-dependencies || exit
+
+	local playwright_test_result=$(npx playwright test)
+
+	if [[ $playwright_test_result == *"1 passed"* ]]
```
Comment (natocesarrego) [inline test_image.sh:241]: @lucasmiranda0, if tomorrow we add more tests, won't this if break? Can you make it a little more robust?

File: test_image.sh [hunk 4]
```diff
@@ -223,6 +224,28 @@ function test_docker_image_scripts_2 {
 	test_page "http://${CONTAINER_HOSTNAME}:${CONTAINER_HTTP_PORT}/test_docker_image_scripts_2.jsp" "TEST2"
 }
 
+function test_docker_image_with_playwright {
```
Comment (natocesarrego) [inline test_image.sh:227]: @lucasmiranda0, can we rename to `test_docker_image_bundle`?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: playwright-dependencies/tests/smoke.spec.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  await page.goto(url);
+
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').fill('test@liferay.com');
+  await page.getByLabel('Email Address').press('Tab');
+  await page.getByLabel('Password').fill('test');
+  await page.getByLabel('Password').press('Tab');
+  await page.getByLabel('Remember Me').press('Tab');
+  await page.getByLabel('Sign In- Loading').getByRole('button', { name: 'Sign In' }).press('Enter');
```
[the remark this answers is not in the document]
Comment (lucasmiranda0) [reply playwright-dependencies/tests/smoke.spec.ts:14]: It just works without the blank space.

File: playwright-dependencies/tests/smoke.spec.ts (hunk 1 above)
Comment (lucasmiranda0) [reply playwright-dependencies/tests/smoke.spec.ts:28]: The variable may be removed, but we still need `await responsePromise;`

**Pull request closed without merging**